.github/workflows/build-and-test.yml

```
@@ -139,3 +139,167 @@ jobs:
+          $env:VERIFY_ACCESS_TOKEN = "false"
+          .\internal\buildscripts\packaging\installer\install.ps1 -access_token "testing123" -msi_path "$msi_path" -mode "${{ matrix.MODE }}"
+          Start-Sleep -s 30
+          Set-Location -Path "$env:ProgramFiles\Splunk\OpenTelemetry Collector"
+          Test-Path -Path ".\splunk-support-bundle.ps1"
+          Start-Process ".\otelcol.exe" -WindowStyle Hidden
+          powershell.exe -File "$env:ProgramFiles\Splunk\OpenTelemetry Collector\splunk-support-bundle.ps1" -t \tmp\splunk-support-bundle
+          Test-Path -Path ".\splunk-support-bundle.zip"
+          Test-Path -Path "\tmp\splunk-support-bundle\logs\splunk-otel-collector.log"
+          Test-Path -Path "\tmp\splunk-support-bundle\logs\splunk-otel-collector.txt"
+          Test-Path -Path "\tmp\splunk-support-bundle\logs\td-agent.log"
+          Test-Path -Path "\tmp\splunk-support-bundle\logs\td-agent.txt"
+          Test-Path -Path "\tmp\splunk-support-bundle\metrics\collector-metrics.txt"
+          Test-Path -Path "\tmp\splunk-support-bundle\metrics\df.txt"
+          Test-Path -Path "\tmp\splunk-support-bundle\metrics\free.txt"
+          Test-Path -Path "\tmp\splunk-support-bundle\metrics\top.txt"
+          Test-Path -Path "\tmp\splunk-support-bundle\zpages\tracez.html"
+          if ( "${{ matrix.MODE }}" -eq "agent" ) {
+            Test-Path -Path "\tmp\splunk-support-bundle\config\td-agent\td-agent.conf"
+            Test-Path -Path "\tmp\splunk-support-bundle\config\agent_config.yaml"
+          }
+          if ( "${{ matrix.MODE }}" -eq "gateway" ) {
+            Test-Path -Path "\tmp\splunk-support-bundle\config\gateway_config.yaml"
+          }
+
+  windows-choco-validation:
+    name: windows-choco-validation
+    runs-on: windows-latest
+    needs: [windows-msi]
+    strategy:
+      matrix:
+        MODE: [ "agent", "gateway" ]
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+
+      - name: Downloading msi build
+        uses: actions/download-artifact@v2
+        with:
+          name: msi-build
+          path: ./dist
+
+      - name: Chocolatey test
+        run: |
+          $ErrorActionPreference = 'Stop'
+          Set-PSDebug -Trace 1
+          $msi_file_name = Resolve-Path .\dist\splunk-otel-collector*.msi | Split-Path -leaf
+          if ($msi_file_name -match '(\d+\.)(\d+\.)+(\d*)') { $version = $matches[0] }
+          write-host "Building choco package..."
+          .\internal\buildscripts\packaging\choco\make.ps1 build_choco -Version $version | Tee-Object -file .\dist\build_logs.log
+          Test-Path -Path ".\dist\splunk-otel-collector.$version.nupkg"
+          write-host "Installing choco package..."
+          choco install splunk-otel-collector -s=".\dist\splunk-otel-collector.$version.nupkg" --params="'/SPLUNK_ACCESS_TOKEN=12345 /MODE:${{ matrix.MODE }}'" -y | Tee-Object -file .\dist\installation_logs.log
```

```suggestion
          choco install splunk-otel-collector -s=".\dist\splunk-otel-collector.$version.nupkg" --params="'/SPLUNK_ACCESS_TOKEN=12345 /SPLUNK_REALM=test /MODE:${{ matrix.MODE }}'" -y | Tee-Object -file .\dist\installation_logs.log
```
Please use a fake realm to not spam our production realm.

---

Yes based on your above suggestion I have checked that default value for realm us0 which would be by default picked up if we don’t specify realm and adding a fake realm would avoid those spamming of the production with the dummy requests. Hence adding a fake realm is a nice idea and I have added that (It was not existing in the CircleCI job but I have added it in the GitHub Action workflow now). Thanks for the suggestion!